modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3404,7 +3402,7 @@ void ONNXImporter::parseQConv(LayerParams& layerParams, const opencv_onnx::NodeP
             padLp.type = "PaddingInt8";
             padLp.set("paddings", DictValue::arrayInt(&paddings[0], paddings.size()));
             padLp.set("depth", CV_8S);
-            padLp.set("value", inp_zp);
+            padLp.blobs.push_back(getBlob(node_proto, 2));
```

Comment:
Not sure that this is correct, because `getBlob(node_proto, 2)` is bias but here is Convolution and `inp_zp` means zero padding (the value after `inp_zp` subtraction). But let's wait for tests (to verify that we cover it)

Reply:
https://github.com/onnx/onnx/blob/main/docs/Operators.md#QLinearConv input of index 2 is the zero point of x (input data).

Reply:
Got it, thanks. Missed that step above:
```cpp
int inp_zp = (int)getScalarFromMat<int8_t>(getBlob(node_proto, 2));
```